### PR TITLE
Add API version 48.0

### PR DESCRIPTION
**Pull request by gnguyen-sfdc**
Adding v48 (Spring '20) support to workbench,

### CODE REVIEW

**Comment by gnguyen-sfdc** [comment]
@ntotten Can you (or anyone else?) review this PR adding support for v48 to workbench? Thank you.

File: workbench/config/defaults.php [hunk 1]
```diff
@@ -321,7 +322,7 @@
     $config["defaultApiVersion"]  = array(
         "label" => "Default API Version",
         "description" => "Default API version to be used for login. This setting does not affect the API version of the current session. Recommended to choose latest version. Some features may act unexpectedly when using older versions.",
-        "default" => "46.0",
+        "default" => "48.0",
```
Comment (praksb) [inline workbench/config/defaults.php]: default version should be one behind the current version. So this should be "47.0"

**Comment by praksb** [comment]
@gnguyen-sfdc I just had one comment. I can approve this review after the small change is made.

**Comment by gnguyen-sfdc** [comment]
@praksb Thank you for reviewing. I made the change to the default API version.

**Pull request merged**